velox/functions/lib/RegistrationHelpers.h

```diff
@@ -83,6 +83,36 @@ void registerUnaryNumeric(const std::vector<std::string>& aliases) {
   registerUnaryFloatingPoint<T>(aliases);
 }
 
+#define REGISTER_NUMERIC_FUNCTIONS(functionName, prefix) \
```

Comment:
Should bool be considered numeric type?

Reply:
No, I find in Spark, bool does not consider as numeric type